docs/SRS/SRS.tex

```diff
@@ -113,9 +129,91 @@ \subsection{Undesired Scenario Handling}
 
 \section{System Level Variables}
 \subsection{Constants}
+
+\begin{table}[H]
+  \centering
+      \setlength{\leftmargini}{0.4cm}
+      \begin{tabular}{| >{\centering\arraybackslash}m{2cm} | 
+        >{\centering\arraybackslash}m{2cm} | 
+        >{\centering\arraybackslash}m{10cm} |}
+      \hline
+      \rowcolor[gray]{0.9}
+      Constant & Unit & Value\\
+      \hline
+      Chess board width & inches & 12\\
+     \hline
+     Chess board length & inches & 12\\
+     \hline
+     Chess board tile width & inches & 1.5\\
+     \hline 
+     Chess board tile length & inches & 1.5\\ 
+     \hline 
+      \end{tabular}
```

Comment:
Should we add a battery voltage in the constants as well or are we just using power outlet as the power source?